### ISSUE TITLE
How To use Old Script :( I don't want to use Baileys-MD 

### ISSUE THREAD

**Issue by gariox3d** [BochilGaming/games-wabot#354]
![Screenshot_2022-06-11-11-09-40-39](https://user-images.githubusercontent.com/82633549/173192673-0eb1d1a8-cac7-46e6-be90-1dea8f18cda7.jpg)

**Comment by BochilGaming** [comment]
maybe you can use modded whatsapp

**Comment by gariox3d** [comment]
> maybe you can use modded whatsapp

Which Mod :(

**Comment by BochilGaming** [comment]
> > maybe you can use modded whatsapp
> 
> Which Mod :(

Idk

**Comment by NugrahaMP** [comment]
iya, saya juga sama masalahnya gini. udh pake wa mod juga. gimana caranya itu?

**Comment by gariox3d** [comment]
> 

Giliran mana yang Anda buat? Bisakah Anda mengiriminya tautan?

**Comment by gariox3d** [comment]
> iya, saya juga sama masalahnya gini. udh pake wa mod juga. gimana caranya itu?

Send me link of WP MOD :)

**Comment by ReyEndymion** [comment]
> > > tal vez puedas usar whatsapp modificado
> > 
> > 
> > que modelo :(
> 
> No sé

Dan mengapa mereka tidak berbicara dengan mereka yang memodifikasi WhatsApp?

Mengapa tidak membuat aplikasi apk untuk bot yang dapat disesuaikan dalam beberapa hal tetapi pengoperasiannya tidak lagi berdasarkan web WhatsApp

**Comment by ReyEndymion** [comment]
> > > > tal vez puedas usar whatsapp modificado
> > > 
> > > 
> > > que modelo :(
> > 
> > 
> > Nariz
> 
> Dan mengapa mereka tidak berbicara dengan mereka yang memodifikasi WhatsApp?
> 
> Mengapa tidak membuat aplikasi apk untuk bot yang dapat disesuaikan dalam beberapa hal tetapi pengoperasiannya tidak lagi berdasarkan web WhatsApp

In conclusion, an APK would reduce connection costs, could be used with several mods and the autoresponders would be direct from the application, obviously without removing the original operation of the Bot

**Comment by DineshValor** [comment]
> > maybe you can use modded whatsapp
> 
> Which Mod :(

His mean, maybe you using whatsapp mod app on which you running bot.
